### ISSUE TITLE
关于patch的size问题

### ISSUE THREAD

**Issue by ShwanMario** [leeyeehoo/CSRNet-pytorch#2]
您好,感谢您百忙之中抽空解答我的疑问.

我有一个问题想请教一下,就是你们在crop 1/4 size的原图的patch后,是直接用patch拿来训练还是resize patch到其他的分辨率后再训练呢?

感谢您的赐教!

**Comment by leeyeehoo** [comment]
直接拿patch做训练

**Comment by markshih91** [comment]
您好,感谢您百忙之中抽空解答我的疑问.我想请教您如下问题：

1、输入图像大小不同，训练时，batch_size只能设置成1吗？
2、或者在这种情况下，怎么时batch_size大于1？

谢谢！！！

**Comment by leeyeehoo** [comment]
只能设成1,或者你自己改一下image.py文件,比如说crop一个fixed的大小的输入.

**Comment by guoqiang01486** [comment]
输入图像大小不同的时候 ,pytorch版本没有问题,但是caffe版本会出现维度不一致的问题.caffe版本输入是不是不能是原始图像,必须crop或resize成8的倍数? 如果test时也必须crop的话, 会不会影响精确度呢?

**Comment by zhengmiao1** [comment]
你好，您的crop好像没有使用呀？因为train.py文件中train=True，而在imge.py中进行crop需要是train在False的情况下

**Comment by ConstantSun** [comment]
> Hello, your crop does not seem to be used? Because train=True in the train.py file, and crop in imge.py needs to be train in the case of False

Yes, I totally agree with @lingkongyingsun  . @leeyeehoo  your crop does not use, in image.py file , you wrote:  if False: ....( which is always not to be executed ) . According to the paper, we need to augment data before training, you did miss that , right @leeyeehoo  ??

**Comment by tomeasure** [comment]
这代码太糟糕了，这几年就不能完善一下吗？
